tests/testthat/test-utils.R

```diff
@@ -29,3 +29,11 @@ testthat::test_that("example_file works", {
 
 })
 
+testthat::test_that("dot_to_package", {
+  p0 <- get_root_path("jsonld")
+  p <- file.path(p0, "R")
+  wd <- setwd(p)
```

Comment:
Would you mind adding a Suggests dependency on withr and using `withr::local_dir()`?